pkgs/tools/admin/trinsic-cli/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, stdenv, rustPlatform, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "trinsic-cli";
+  version = "1.1.0";
+
+  src = fetchurl {
+    url = "https://github.com/trinsic-id/sdk/releases/download/v${version}/${pname}-1.1.2.tar.gz";
```

Comment:
Is there anything the release tarball offers over the raw source? Also, do you happen to know what the -nix tarball is?

Reply:
Custom `build.rs` file removed, which depends on `protoc` compiler - only used as dev tool. The languages for other bindings are excluded in the `-nix` tarball and also a missing Cargo.lock was included. I uploaded some manual packages that fix these issues.